code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -83,13 +83,22 @@
 
 	air.volume = volume
 
+	/**
+	 *  For a machinery to properly "connect" to a pipenet and share gasses, 
+	 *  the pipenet needs to acknowledge a gas mixture as it's member.
+	 *  This is currently handled by the other_airs list in the pipeline datum.
+	 *  
+	 *	Other_airs itself is populated by gas mixtures through the parents list that each machineries have.
+	 *	This parents list is populated when a machinery calls update_parents and is then added into the queue by the controller.
+	 */
```

Comment:
Minor nitpicks, the pipenet stuff eh, I think the misnomer might be my fault, but I may also be giving myself too much credit
```suggestion
	/**
	 *  For a machine to properly "connect" to a pipeline and share gases, 
	 *  the pipeline needs to acknowledge a gas mixture as it's member.
	 *  This is currently handled by the other_airs list in the pipeline datum.
	 *  
	 *	Other_airs itself is populated by gas mixtures through the parents list that each machineries have.
	 *	This parents list is populated when a machinery calls update_parents and is then added into the queue by the controller.
	 */
```